tests/phpunit/tests/html-api/wpHtmlTagProcessor-internals.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Unit tests covering WP_HTML_Tag_Processor internal helpers.
+ *
+ * @package WordPress
+ * @subpackage HTML-API
+ */
+
+/**
+ * @group html-api
+ *
+ * @coversDefaultClass WP_HTML_Tag_Processor
+ */
+class Tests_HtmlApi_WpHtmlTagProcessor_Internals extends WP_UnitTestCase {
+	/**
+	 * @ticket {TICKET NUMBER}
```

Comment:
```suggestion
	 * @ticket 59291
```

Reply:
Thanks for copying that here! Replaced in 7151083918